packages/sdk/src/modules/liquidation/redemptionStrategy.ts

```diff
@@ -229,6 +230,44 @@ const getStrategyAndData = async (fuse: MidasBase, inputToken: string): Promise<
         outputToken,
       };
     }
+    case RedemptionStrategyContract.ERC4626Liquidator:
+      let fee: number;
+      let underlyingTokens: string[];
+
+      switch (inputToken) {
+        case underlying(ethereum.assets, assetSymbols.realYieldUSD): {
+          fee = 10;
+          underlyingTokens = [
+            underlying(ethereum.assets, assetSymbols.USDC),
+            underlying(ethereum.assets, assetSymbols.DAI),
+            underlying(ethereum.assets, assetSymbols.USDT),
+          ];
+        }
+        case underlying(ethereum.assets, assetSymbols.ethBtcMomentum):
+        case underlying(ethereum.assets, assetSymbols.ethBtcTrend): {
+          underlyingTokens = [
+            underlying(ethereum.assets, assetSymbols.USDC),
+            underlying(ethereum.assets, assetSymbols.WETH),
+            underlying(ethereum.assets, assetSymbols.WBTC),
+          ];
+          fee = 500;
+        }
+        default: {
+          fee = 1000;
+          underlyingTokens = [inputToken];
```

Comment:
`underlyingTokens = [outputToken];` seems more correct to me

Reply:
the default swap fee is 10%, is that a common case?

Reply:
The others had to be manually decided by testing out the liquidations in forge. With different fee parameters, liquidations fail. So that should be the modus operandi anyways, always validate in forge the pool dees